Line/iOS/LINELoginDemo/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'LINELoginDemo' do
+  # Uncomment this line if you're using Swift or would like to use dynamic frameworks
+  # use_frameworks!
+
+  # Pods for LINELoginDemo
+  pod 'Firebase/Auth', '3.6.0'
```

Comment:
Curious why you're locking it to this version number -- we're already up to Firebase 3.7.1, and it seems like we probably want our samples to be the same. If you're not locking this to 3.6.0 for any particular reason, it might be best to just leave out the version numbers so we always grab the latest.